### PR TITLE
added solution.1

### PR DESCRIPTION
**Pull request by VitaliiLeshchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,51 @@
  */
 
 public class Plane {
+    private int param1;
+    private int param2;
+    private int param3;
+    private int param4;
+    private int param5;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: not informative naming

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,51 @@
...
+    public Plane(PlaneBuilder planeBuilder) {
+
+    }
+
+    public class PlaneBuilder {
+
+        private int paramParam;
+
+        public PlaneBuilder(int param1) {
+            Plane.this.param1 = param1;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: should builder have any constructors?

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,51 @@
...
 
 public class Plane {
+    private int param1;
+    private int param2;
+    private int param3;
+    private int param4;
+    private int param5;
 
+    public Plane(PlaneBuilder planeBuilder) {
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java:10]: why is this constructor empty?

Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Plane.java:10]: done

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Plane.java]: i did rewrite whole solution.
public static void main(String[] args) {
        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder();
        planeBuilder.setName("AirPlane");

        System.out.println();//blaBla business logic
        System.out.println();//blaBla business logic

        Plane plane = planeBuilder
                .setHeight(1)
                .setModel("Model")
                .setWeight(2)
                .setAge(999)
                .build();
    }
now it work

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**